$NetBSD: patch-aa,v 1.3 2026/01/04 13:35:48 wiz Exp $

pcap-int.h does not exist any longer.
Avoid conflict with libpcap.

--- pcaputil.c.orig	2000-11-14 15:51:08.000000000 +0000
+++ pcaputil.c
@@ -14,6 +14,7 @@
 #include <string.h>
 #include <err.h>
 #include <pcap.h>
+#undef BSD
 #ifdef BSD
 #include <pcap-int.h>
 #endif
@@ -56,7 +57,7 @@ pcap_dloff(pcap_t *pd)
 }
 
 pcap_t *
-pcap_init(char *intf, char *filter, int snaplen)
+mypcap_init(char *intf, char *filter, int snaplen)
 {
 	pcap_t *pd;
 	u_int net, mask;
